generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management-dialog.component.ts

```diff
@@ -120,15 +120,15 @@ export class <%= entityAngularName %>DialogComponent implements OnInit {
         return this.dataUtils.openFile(contentType, field);
     }
 
-    setFileData(event, <%= entityInstance %>, field, isImage) {
+    setFileData(event, entity, field, isImage) {
```

Comment:
@ruddell I guess this method could be moved into dataUtils as well as its generic now